tests/node/rpc/messages/UnsubscribeVehicleDataResponseTests.js

```diff
@@ -29,12 +35,17 @@ describe('UnsubscribeVehicleDataResponseTests', function () {
             .setDataType(VehicleDataType.VEHICLEDATA_WINDOWSTATUS);
         const JSON_WINDOWSTATUS = this.windowStatus.getParameters();
 
+        this.climateData = new VehicleDataResult()
+          .setDataType(VehicleDataType.VEHICLEDATA_CLIMATEDATA);
+        const JSON_CLIMATEDATA = this.climateData.getParameters();
+
         this.createMessage = function () {
             return new UnsubscribeVehicleDataResponse()
                 .setStabilityControlsStatus(this.stabilityControlsStatus)
                 .setHandsOffSteering(this.vehicleDataResult)
```

Comment:
**Optional:** On line 45 you can change `vehicleDataResult` to `handsOffSteering`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.